lib/TimeZoneDate.js

```diff
@@ -0,0 +1,239 @@
+(function() {
```

Comment:
Cheers for the PR. The built source code shouldn't be committed to source. I ignored the lib/ directory in gitignore. When publishing it will build the package and running `npm install tz-date` will include the lib directory.